portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the value entered in the form.
+    String textValue = request.getParameter("text-input");
+    // Print the value so you can see it in the server logs.
+    System.out.println("You submitted: " + textValue);
+    // Write the value to the response so the user can see it.
+    response.getWriter().println("You submitted: " + textValue);
```

Comment:
The use won't see this message since you redirect immediately below....

Reply:
Removed the redirect, was just for testing purposes and the redirect works.

Reply:
Point noted and fixed.

Reply:
@acheng1 https://rkyatham-sps-summer22.uc.r.appspot.com/ 
Here is the link to it, all the changes requested are fixed.